packages/sveltekit/src/vite/sentryVitePlugins.ts

```diff
@@ -37,7 +37,7 @@ const DEFAULT_PLUGIN_OPTIONS: SentrySvelteKitPluginOptions = {
  * Sentry adds a few additional properties to your Vite config.
  * Make sure, it is registered before the SvelteKit plugin.
  */
-export function sentrySvelteKit(options: SentrySvelteKitPluginOptions = {}): Plugin[] {
+export async function sentrySvelteKit(options: SentrySvelteKitPluginOptions = {}): Promise<Plugin[]> {
```

Comment:
This is technically breaking as it changes the return type of the function but the good news is that noone has to await this function as the vite config is perfectly fine with taking a `Promise<Plugin[]>`. I should have done this already but for some reason decided to stay sync in #7811. I vote we go forward with making it async, not just because we need it for this change but also because it gives us more freedom in the future.